docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -466,6 +740,53 @@ CompletableFuture<Void> allInsertsFuture =
 allInsertsFuture.thenRun(() -> System.out.println("Seeded transaction events."));
 ```
 
+Create the anomalies tables:
+
+```java
+String sql = "CREATE TABLE possible_anomalies WITH ("
+             + "    kafka_topic = 'possible_anomalies',"
+             + "    VALUE_AVRO_SCHEMA_FULL_NAME = 'io.ksqldb.tutorial.PossibleAnomaly'"
+             + ")   AS"
+             + "    SELECT card_number AS `card_number_key`,"
+             + "           as_value(card_number) AS `card_number`,"
+             + "           latest_by_offset(email_address) AS `email_address`,"
+             + "           count(*) AS `n_attempts`,"
+             + "           sum(amount) AS `total_amount`,"
+             + "           collect_list(tx_id) AS `tx_ids`,"
+             + "           WINDOWSTART as `start_boundary`,"
+             + "           WINDOWEND as `end_boundary`"
+             + "    FROM transactions"
+             + "    WINDOW TUMBLING (SIZE 30 SECONDS)"
+             + "    GROUP BY card_number"
+             + "    HAVING count(*) >= 3"
+             + "    EMIT CHANGES;";
+Map<String, Object> properties = Collections.singletonMap("auto.offset.reset", "earliest");
+client.executeStatement(sql, properties).get();
+```
+
+Check contents of the anomalies table with a push query:
```

Comment:
This example currently suffers from the same issue as the regular (CLI-based) version of the tutorial, and will need the corresponding update applied: https://github.com/confluentinc/ksql/issues/5827

Reply:
Updated the PR with the same fix as in https://github.com/confluentinc/ksql/pull/5840, so this is now resolved.